content/en/docs/how-tos/add-jobs-to-testgrid.md

```diff
@@ -0,0 +1,39 @@
+* `Informing jobs` are a broader suite that test the variety of enviroments and configurations our customers expect. 
+* `Broken jobs` are those that have a known, triaged failure that prevents their function for a sustained period of time (more than a week).
+
+## How to decide if a job should qualify as a release-informing job?
+Release informing jobs in the [release controller config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) get run every time we build a new payload for that release.
+So you need to ask yourself whether the given job:
+1) is reliable enough to provide a useful signal when it fails
+2) covers an important function that determines whether the payload is viable or not, which is not covered by other informing or blocking jobs
+
+If it does not meet the qualifications for being a release controller informing job, it can still be added to testgrid (and thus [sippy](https://sippy.ci.openshift.org/)) by adding it to the [_allow-list](https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml). In which case it won't be run against every new payload that is built, it would instead just run on whatever periodic schedule you configured it for.
+
+## How are jobs classified as release-informing or blocking?
+The [release config](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases) and the `optional` job annotation combine to determine the testgrid dashboard. If a job in the release definition is an upgrade job it goes into the overall informing dashboard (because upgrades cross dashboards). If it is optional it is considered informing, and is otherwise considered blocking. 
+
+
+## How do I add a job to testgrid?
+Release gating jobs are configured via the `verify` property in [_releases/release-*.json](https://github.com/openshift/release/tree/master/core-services/release-controller/_releases), with the `optional` property distinguishing between blocking and informing jobs.
+
+Changes should automatically be picked up by the [periodic-prow-auto-testgrid-generator](https://prow.ci.openshift.org/?job=periodic-prow-auto-testgrid-generator) job which runs daily and a PR would be created to merge these changes into [kubernetes/test-infra/config/testgrids/openshift](https://github.com/kubernetes/test-infra/tree/master/config/testgrids/openshift). 
+If it is not merged in a timely manner 
+1. Search for a PR titled [Update OpenShift testgrid definitions by auto-testgrid-generator job](https://github.com/kubernetes/test-infra/pulls?q=is%3Apr+is%3Aopen+Update+OpenShift+testgrid+definitions+by+auto-testgrid-generator+job) in [kubernetes/test-infra](https://github.com/kubernetes/test-infra)
+2. Request the assignee to approve the PR
+
+### Overriding the default behavior in testgrid
```

Comment:
You can't override anything with the allow-list, you can only additively amend it.